java-client/src/main/java/org/scassandra/http/client/PrimingClient.java

```diff
@@ -185,7 +191,7 @@ private void httpDelete(String url) {
         }
     }
 
-    private void prime(PrimingRequest primeRequest, String url) {
+    private void prime(Object primeRequest, String url) {
```

Comment:
Is there some kind of shared interface we can make between PrimingRequest and BatchPrimingRequest to prevent user from giving some kind of invalid input?    It looks like there is no shared explicit contract needed since gson is used for serialization, but would be nice to have something for clarity anyways.